compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3635,8 +3635,14 @@ TR::Register *OMR::Power::TreeEvaluator::vnegEvaluator(TR::Node *node, TR::CodeG
 
    switch(node->getDataType().getVectorElementType())
      {
+     case TR::Int8:
+       return TR::TreeEvaluator::vnegIntHelper(node,cg,TR::Int8);
```

Comment:
Please add a space between the arguments like `(node, cg, TR::Int8)`.
See `vsubEvaluator()` above, for example.